### PR TITLE
SineReLU activation function added

### PR DESCRIPTION
**Pull request by sleepingcat4**
I have noticed in the activations.py file, SineReLU was not present. Though another variant leaky relu was present. leaky variant is better for a wide variety of tasks but in certain applicants of image processing and natural language sinerelu produces superior results. 

- Prevents dead neuron syndrome
- easy to compute 
- Shows amazing capabilities in image & NLP tasks. And efficent too.

### CODE REVIEW

**Comment by sleepingcat4** [comment]
fixed indentation error in the activation.py file

File: praxis/layers/activations.py [hunk 1]
```diff
@@ -67,7 +67,14 @@ def __call__(self, inputs: JTensor) -> JTensor:
     """Applies the activation function."""
     return jax.nn.relu(inputs)
 
+class sineReLU(BaseActivation):
```
Comment (zhangqiaorjc) [inline praxis/layers/activations.py:70]: nit: do you mind capitalizing sineReLU following naming convention in the rest of the file?

**Review by zhangqiaorjc — APPROVED** [review]
thanks! sorry for the late review

File: praxis/layers/activations.py (hunk 1 above)
Comment (sleepingcat4) [reply praxis/layers/activations.py:70]: Sure! Doing this in a moment

**Comment by sleepingcat4** [comment]
I have captialised the sineReLU to SINERELU

**Pull request merged**